account_invoice_force_removal/tests/test_move.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.exceptions import UserError
+from odoo.tests import Form
+from odoo.tests.common import SavepointCase
+
+
+class TestMOve(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Test customer", "customer_rank": 1}
+        )
+        cls.journal = cls.env["account.journal"].create(
+            {
+                "name": "Test journal",
+                "type": "sale",
+                "code": "test-sale-jorunal",
+                "company_id": cls.env.company.id,
+            }
+        )
+        cls.product = cls.env["product.product"].create(
+            {"name": "Test product", "type": "service"}
+        )
+        invoice = Form(
+            cls.env["account.move"].with_context(
+                default_type="out_invoice", default_company_id=cls.env.company.id
+            )
+        )
+        invoice.partner_id = cls.partner
+        invoice.journal_id = cls.journal
+        with invoice.invoice_line_ids.new() as line_form:
+            line_form.name = cls.product.name
+            line_form.product_id = cls.product
```

Comment:
If you want, for invoices the product is not mandatory, so we can save code lines and you do not need create a product

Reply:
Although `product_id` isn't mandatory but help to auto-define other fields (`account_id` for example).

Reply:
Yes!! but account_id if the has not a product Odoo get it from Journal Default Credit Account